### DISCUSSION TITLE
Parallel downloading like Edge/Chrome

### DISCUSSION THREAD

**Discussion by gunir** [Floorp-Projects/Floorp#441] [Feature Request]
Parallel downloading is a feature of Edge/Chrome/Chromium-based browsers to download file using multi-threaded technology (Internet Download Manager uses it) by splitting download content into multiple smaller parts and download at the same time which greatly improve download speed in cases:

- Websites limit download speed by a single connection
- ISPs limit download speed by a single connection

`chrome://flags#chrome-parallel-download`

![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/fdef1d80-5053-47b9-9464-abd871a3531a)

**Comment by creeper-0910** [comment]
Use extensions.

**Comment by creeper-0910** [comment]
PS
Parallel downloads should already be enabled in Firefox and its derivatives.
